exporter/debugexporter/internal/otlptext/databuffer.go

```diff
@@ -325,7 +325,17 @@ func (b *dataBuffer) logProfileSamples(ss pprofile.SampleSlice) {
 			}
 		}
 		b.logEntry("        Value: %d", sample.Value().AsRaw())
-		b.logEntry("        Attributes: %d", sample.Attributes().AsRaw())
+
+		if lai := sample.Attributes().Len(); lai > 0 {
+			b.logEntry("        Attributes:")
+			for j := 0; j < lai; j++ {
+				if sample.Attributes().At(j) <= math.MaxInt32 {
```

Comment:
The next version of the proto uses an `int32` rather than an `uint64` here. So after the next proto release/upgrade, we won't need this check anymore.